### PR TITLE
Removed html from init

### PR DESCRIPTION
**Pull request by daavoo**
P.R. for https://github.com/iterative/dvclive/pull/133

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
What about https://dvc.org/doc/dvclive/api-reference/next_step @daavoo, is `html` still available there?

https://dvc.org/doc/dvclive/user-guide/dvclive-with-dvc#html-report also mentions it inside the `live` config section of dvc.yaml. Is it available there? E.g. with `stage add --live-no-html`

Thanks

**Comment by daavoo** [comment]
> What about https://dvc.org/doc/dvclive/api-reference/next_step @daavoo, is `html` still available there?

Good catch, that also needs to be updated.

> https://dvc.org/doc/dvclive/user-guide/dvclive-with-dvc#html-report also mentions it inside the `live` config section of dvc.yaml. Is it available there? E.g. with `stage add --live-no-html`

Yes it is still available on the "DVC side" (`run`/`stage add` commands and `dvc.yaml`). We removed `html` from the "Python API" (`dvclive.init`) mainly because it is an option that can only work when using DVC. `stage add --live <path>` will activate the html generation via ENV vars.

**Review by pared — APPROVED** [review]

File: content/docs/dvclive/api-reference/init.md [hunk 1]
```diff
@@ -45,11 +43,6 @@ all existing `.tsv` files, `{path}.json` and `{path.html}`.
   call, DVCLive will generate a _metrics summary_ (usable by `dvc metrics`). The
   _summary_ will be located at `{path}.json`.
 
-- `html` (`True` by default) - works only when DVCLive is used alongside DVC. If
```
Comment (dberenbaum) [inline content/docs/dvclive/api-reference/init.md:48]: There's still a note above about cleanup removing `{path.html}`. I think this is still technically correct, but it seems a bit confusing without any other mention of html files.

File: content/docs/dvclive/api-reference/next_step.md [hunk 2]
```diff
@@ -41,13 +41,15 @@ The _metrics summary_ will be saved to `{path}.json`. Here's an example:
...
 
-By default, on each `dvclive.next_step()` call, `DVC` will create a new
-[checkpoint](/doc/user-guide/experiment-management/checkpoints).
+By default, on each `dvclive.next_step()` call, `DVC` will prepare an HTML
+report with all the _metrics logs_ logged in `path`.
 
-In addition, if `html` is True, on each `dvclive.next_step()` call, `DVC` will
-prepare an HTML report with all the _metrics logs_ logged in `path`.
+When [checkpoints](<(/doc/user-guide/experiment-management/checkpoints)>) are
+enabled in the <abbr>pipeline</abbr> , `DVC` will create a new _checkpoint_ on
```
Comment (jorgeorpinel) [inline content/docs/dvclive/api-reference/next_step.md:51]: Copy edit
```suggestion
When [checkpoints](/doc/user-guide/experiment-management/checkpoints) are
enabled in the <abbr>pipeline</abbr>, `DVC` will create a new _checkpoint_ on
```

File: content/docs/dvclive/api-reference/next_step.md [hunk 3]
```diff
@@ -41,13 +41,15 @@ The _metrics summary_ will be saved to `{path}.json`. Here's an example:
 ### DVC Integration
 
 When `dvclive` is used alongside `DVC`, each `dvclive.next_step()` call will
-have additional features.
+have additional features. See
+[DVCLive with DVC](/doc/dvclive/user-guide/dvclive-with-dvc).
```
Comment (jorgeorpinel) [inline content/docs/dvclive/api-reference/next_step.md:45]: https://dvc.org/doc/dvclive/user-guide/dvclive-with-dvc doesn't explain anything about `dvclive.next_step()`. It's just used in a code block. Maybe avoid this change  for now since it's out of scope anyway.

**Comment by dberenbaum** [comment]
A couple other mentions I noticed:
* https://dvc.org/doc/dvclive/user-guide/dvclive-with-dvc: Description of `--live-no-html` needs to be updated (maybe these should give short explanations instead of pointing back to the api ref anyway).
* https://dvc.org/doc/command-reference/run: Same.
* https://dvc.org/doc/command-reference/stage/add: Same.

**Comment by daavoo** [comment]
> A couple other mentions I noticed:
> 
>     * https://dvc.org/doc/dvclive/user-guide/dvclive-with-dvc: Description of `--live-no-html` needs to be updated (maybe these should give short explanations instead of pointing back to the api ref anyway).
> 
>     * https://dvc.org/doc/command-reference/run: Same.
> 
>     * https://dvc.org/doc/command-reference/stage/add: Same.

Good points. I will update those

File: content/docs/command-reference/run.md [hunk 4]
```diff
@@ -270,15 +270,19 @@ $ dvc run -n second_stage './another_script.sh $MYENVVAR'
...
+  [DVCLive](/doc/dvclive/user-guide/dvclive-with-dvc) to write logs in. `path`
+  will be tracked (<abbr>cached</abbr>) by DVC. Saved in the `live` field of
+  `dvc.yaml`.
 
-- `--live-no-cache <path>` - the same as `-o` except that the `path` is not
+- `--live-no-cache <path>` - the same as `--live` except that the `path` is not
   tracked by DVC. Useful if you prefer to track it with Git.
 
-- `--live-no-summary` - passes `summary=False` to Dvclive config.
+- `--live-no-summary` - deactivates DVCLive
```
Comment (daavoo) [inline content/docs/command-reference/run.md:280]: This is not strictly related to the html change but I think it just make sense to be consistent

**Review by dberenbaum — APPROVED** [review]
LGTM!

File: content/docs/command-reference/run.md [hunk 5]
```diff
@@ -270,15 +270,19 @@ $ dvc run -n second_stage './another_script.sh $MYENVVAR'
   affect any DVC operations.
 
 - `--live <path>` - specify the directory `path` for
-  [Dvclive](/doc/dvclive/dvclive-with-dvc) to write logs in. `path` will be
-  tracked (<abbr>cached</abbr>) by DVC. Saved in the `live` field of `dvc.yaml`.
+  [DVCLive](/doc/dvclive/user-guide/dvclive-with-dvc) to write logs in. `path`
```
Comment (jorgeorpinel) [inline content/docs/command-reference/run.md:273]: BTW (unrelated) did we ever setup redirects for these link changes? Check /redirects-list.json

File: content/docs/dvclive/user-guide/dvclive-with-dvc.md [hunk 6]
```diff
@@ -94,17 +95,17 @@ with `dvc exp show`/`dvc exp diff`.
...
+[compare and visualize](/doc/studio/user-guide/visualize-experiments)
 experiments using DVCLive in DVC Studio.
 
 ![](/img/dvclive-studio-plots.png)
 
 ### HTML report
 
 In addition to the
-[outputs described in the Quickstart](/docs/dvclive/user-guide/quickstart#outputs),
+[outputs described in the Quickstart](/doc/dvclive/user-guide/quickstart#outputs),
```
Comment (jorgeorpinel) [inline content/docs/dvclive/user-guide/dvclive-with-dvc.md:108]: Unrelated?

File: content/docs/dvclive/user-guide/quickstart.md [hunk 7]
```diff
@@ -65,18 +65,24 @@ dvclive        train.py
...
 timestamp	step	{metric_name}
 1614129197192	0	0.7612833380699158
 1614129198031	1	0.8736833333969116
 1614129198848	2	0.8907166719436646
 ```
 
-And also a _summary_ with the latest metrics:
+### Metrics Summary
+
+In addition, when [`summary`](/doc/dvclive/api-reference/init#parameters) is
```
Comment (jorgeorpinel) [inline content/docs/dvclive/user-guide/quickstart.md:83]: Unrelated?

**Review by jorgeorpinel — APPROVED** [review]
LGTM as well.

> Please try to make separate PRs for unrelated changes as a best practice to make reviews easier/faster. Thanks

File: content/docs/command-reference/run.md (hunk 5 above)
Comment (daavoo) [reply content/docs/command-reference/run.md:273]: Didn't know about that, thanks!

Comment (daavoo) [reply content/docs/command-reference/run.md:273]: You are refering to https://github.com/iterative/dvc.org/blob/master/content/linked-terms.js , right?

Comment (jorgeorpinel) [reply content/docs/command-reference/run.md:273]: https://github.com/iterative/dvc.org/blob/master/redirects-list.json

Comment (jorgeorpinel) [reply content/docs/command-reference/run.md:273]: We use it when docs have been published earlier. Doing a redirect helps search engines transfer ranking to the new URL. Also if anyone linked the old URL from their blogs or something, this way their links won't brake.

Comment (daavoo) [reply content/docs/command-reference/run.md:273]: Oh, got it!
